worldedit-core/src/main/java/com/sk89q/worldedit/internal/util/DeprecationUtil.java

```diff
@@ -44,7 +43,7 @@ private DeprecationUtil() {
      */
     public static void checkDelegatingOverride(Class<?> implementingClass) {
         // pull the information about the caller
-        StackTraceElement caller = Throwables.lazyStackTrace(new Throwable()).get(1);
```

Comment:
This and the change in `LogManagerCompat` could make use of `StackWalker` instead

Reply:
Yes, we should 100% be doing that instead.